core/site_scons/models/T3T1/emulator.py

```diff
@@ -43,6 +44,7 @@ def configure(
 
     if "sd_card" in features_wanted:
         features_available.append("sd_card")
+        defines += ["USE_SD_CARD=1"]
```

Comment:
Shouldn't USE_SD_CARD=1 be one level higher?

Reply:
yes https://github.com/trezor/trezor-firmware/commit/ca7850be3a05da9ef650dc38117491ad9f5c0bd3